templates/default/rt/rtticketeventlist.html

```diff
@@ -13,8 +13,10 @@
 			{if $ticket.state != $smarty.const.RT_RESOLVED}
 				{button icon="add" label="Add"
 					onclick="location.href='?m=eventadd&ticketid={$ticket.ticketid}{if $ticket.customerid}&customerid={$ticket.customerid}{/if}';"}
-				{button icon="close" label="Close all"
-					onclick="location.href='?m=eventedit&action=close&ticketid={$ticket.ticketid}';"}
+				{if $openeventcount}
+					{button icon="close" label="Close all"
```

Comment:
Tak jeszcze zapytam, gdzie jest ta zmienna Smarty zdefiniowana? Nie mogę się doszukać, ani w master, ani stable, ani stable-24 :(

Reply:
To nie jest krytyczny błąd. 

Są dwie opcje:
- przenosimy zmianę związaną z liczeniem tej zmiennej w funkcji GetQueueContents()
- olewamy

Moja propozycja jest ta druga.

Reply:
Jeśli to button tylko przejśćiem bez potwierdzania to można użyć w `{button ...}` parametry `href="..."` bezpośrednio (niedawno wprowadzone - przypominam).
No chyba, że to ma być przeniesione do którychś gałęzi innych?